### DISCUSSION TITLE
New more blog styles for your Category Blog

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#643] [Ideas]
I am working on new blog styles for your Category Blog. 

- List Left
- List Right
- Zigzag
- Image Bottom

Which do you prefer?
![Screen Shot 2024-04-10 at 11 01 23](https://github.com/templaza/astroid-framework/assets/3204103/0ed9ceb5-0dfd-4a11-a350-436af7e09506)
![Screen Shot 2024-04-10 at 11 01 47](https://github.com/templaza/astroid-framework/assets/3204103/67d64c22-8627-48a1-80a5-38e67be8bd64)
![Screen Shot 2024-04-10 at 11 02 03](https://github.com/templaza/astroid-framework/assets/3204103/12459a50-21a3-446d-81ba-55ab09f296d2)
![Screen Shot 2024-04-10 at 11 02 46](https://github.com/templaza/astroid-framework/assets/3204103/caf9ff5d-313e-4600-a6dd-528dc89bea4b)

### HOW TO TEST?

1. Download and install v3-branch. 
2. Clear Astroid Cache to recompile scss
3. Go to edit your Content Menu -> Category Blog type -> Blog Options
4. Select your Item Image postion and try to configure Image width options
5. Save and Go to front-end

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
I will try tomorrow..... Need to go to bed.... 😴
I find them all good and interesting as different design choices.

**2 QUESTIONS**
**1• The following are tags, Article badges, others????**
![tags](https://github.com/templaza/astroid-framework/assets/1907276/fff6d4a1-5224-473f-97ec-3114a45f71e2)

I ask, because one thing that can be useful for a blog is to have each article showing their Joomla category in a nice badge (e.g. top right square or pill element with category article name.). This element can even be slightly out of the article boundary. It is nice to have because companies or people blog can have different types of content (e.g. News, Special, Events, Coming Soon etc....)

A bit like https://grafcomm.ca/en/graf-blog but here the date is shown as article badge. **(This is another possibility that can be great)** — Both the date or Article categories should have a prominent spot on blogs, I think.

**2• Not sure about those????**
![Screen Shot 2024-04-10 at 1 33 13 AM](https://github.com/templaza/astroid-framework/assets/1907276/f51d6e0e-e96c-4004-ac91-8348c2e35866)

**Comment by vtheod** [comment]
They are all nice.
Closest to my taste is the list right and the zig zag.
Regarding the image bottom, I wouldn't hide the title with the bagde. I would place it over the image (like you do to the other layouts).

**Comment by ghost** [comment]
I think everyone is beautiful. It would be great if you could choose between everyone. Just like you need it on your website.

**Reply by sonvnn** [reply]
Yes, you can configure it
![Screen Shot 2024-04-12 at 08 28 41](https://github.com/templaza/astroid-framework/assets/3204103/16547253-894e-4ff1-b308-d5189ba82579)

**Reply by Chacapamac** [reply]
**This is really cool !!!!!!** 🚀
I don’t remember of any framework that permit users to **easily control the look over the so important blog Joomla feature**.

**Where do you find these settings?**

**Reply by sonvnn** [reply]
> **This is really cool !!!!!!** 🚀
> 
> I don’t remember of any framework that permit users to **easily control the look over the so important blog Joomla feature**.
> 
> 
> 
> **Where do you find these settings?**

1. Go to menu manager -> new an article menu -> choose category blog type 
2. In Blog Options tab you can find these options

**Comment by Chacapamac** [comment]
Is this suppose to show on Joomla ‎4.4.3 — Astroid 3.0.13 ?

**Reply by sonvnn** [reply]
You can download Astroid 3.0.14-rc1 here https://github.com/templaza/astroid-framework/releases/tag/v3.0.14-rc1
Blog Options is working with Two template only. I will update Zero and One in shorttime.

**Comment by ghost** [comment]
I think this is only included in Astroid 3.0.14 and in V3 Brunch. I will install V3 Brunch tomorrow and look at the blog option. But I think it will be great.

**Reply by sonvnn** [reply]
You can download Astroid 3.0.14-rc1 here https://github.com/templaza/astroid-framework/releases/tag/v3.0.14-rc1
Blog Options is working with Two template only. I will update Zero and One in shorttime.

**Reply by deltapapa01** [reply]
Hi Sonny,
did you know, when you can update Zero to the new Blog Style? I want to make a Sponsor Site for a customer, and the new blog style is perfect. I don't want to configure my site with TWO complett new, and export and import is not working jet.

WBR,
deltapapa

**Reply by sonvnn** [reply]
@deltapapa01 Thank you! I will update it asap. I am so crazy busy with my company project at the moment. :(
